virttest/utils_test/libvirt.py

```diff
@@ -2092,6 +2092,51 @@ def create_nwfilter_xml(params):
         raise exceptions.TestFail("Fail to create nwfilter XML: %s" % detail)
 
 
+def get_channel_info(session, interface):
+    """
+    Get channel parameters of interface in vm.
+    """
+    cmd = "ethtool -l %s" % interface
+    s, o = session.cmd_status_output(cmd)
+    if s:
+        logging.info("Get channel parameters for vm failed:%s" % o)
+        return {}, {}
+    maximum = {}
+    current = {}
+    # Just for temp
+    settings = {}
+    for line in o.splitlines():
+        if line.count("maximums"):
+            settings = maximum
+        elif line.count("Current"):
+            settings = current
+        parameter = line.split(':')[0].strip()
+        if parameter in ["RX", "TX", "Other", "Combined"]:
+            settings[parameter] = line.split(':')[1].strip()
+    return maximum, current
+
+
+def setting_channel(session, interface, parameter, value):
+    """
+    Setting channel parameters for interface.
+    """
```

Comment:
Complete the docstring for params and return

Reply:
updated, Thank you!